src/Symfony/Bundle/TwigBundle/composer.json

```diff
@@ -23,7 +23,7 @@
         "symfony/twig-bridge": "^6.4|^7.0",
         "symfony/http-foundation": "^6.4|^7.0",
         "symfony/http-kernel": "^6.4|^7.0",
-        "twig/twig": "^3.0.4"
+        "twig/twig": "^3.11"
```

Comment:
@fabpot Is that the correct way ? Because I'm not sure there's already a Twig release with the new Caches.

Reply:
This is fine IMO. Symfony 7.2 will be release in November. We will definitely have a Twig release tagged before that date.